### PR TITLE
Improvements to URL Details Endpoint (Trac 54358)

### PR DESCRIPTION
**Pull request by hellofromtonya**
Follow-up to changeset https://core.trac.wordpress.org/changeset/51973 to improve:

* Translations by removing HTML tags and spelling out `OG`
* Adding `@since` param to new filters
* Improving comments for code standards and consistency
* Improving readability by making multiple args multiline
* Micro-optimizations to avoid unnecessary variable assignments

Trac ticket: https://core.trac.wordpress.org/ticket/54358

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by swissspidy — APPROVED** [review]
LGTM

**Comment by hellofromtonya** [comment]
Thank you @swissspidy! 

Hey @SergeyBiryukov, Could you please review the PR? Thanks!

File: src/wp-includes/rest-api/endpoints/class-wp-rest-url-details-controller.php [hunk 1]
```diff
@@ -200,8 +217,9 @@ public function permissions_check() {
 	 *
 	 * @since 5.9.0
 	 *
-	 * @param string $url The website url whose HTML we want to access.
-	 * @return string|WP_Error The HTTP response from the remote URL, or an error.
+	 * @param string $url The website url whose HTML to access.
```
Comment (SergeyBiryukov) [inline src/wp-includes/rest-api/endpoints/class-wp-rest-url-details-controller.php:220]: Let's capitalize "URL" consistently here :)

**Review by SergeyBiryukov — APPROVED** [review]
This looks great to me, thanks for working on it!

Left one minor comment, should be good to go otherwise.

File: src/wp-includes/rest-api/endpoints/class-wp-rest-url-details-controller.php (hunk 1 above)
Comment (hellofromtonya) [reply src/wp-includes/rest-api/endpoints/class-wp-rest-url-details-controller.php:220]: Fixed in commit. Thank you @SergeyBiryukov!

**Comment by hellofromtonya** [comment]
Committed via changeset https://core.trac.wordpress.org/changeset/52008.

**Pull request closed without merging**